cli/compose/schema/bindata.go

```diff
@@ -10,27 +10,28 @@ package schema
 import (
 	"bytes"
 	"compress/gzip"
-	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
 	"time"
+
+	"github.com/pkg/errors"
 )
 
 func bindataRead(data []byte, name string) ([]byte, error) {
 	gz, err := gzip.NewReader(bytes.NewBuffer(data))
 	if err != nil {
-		return nil, fmt.Errorf("Read %q: %v", name, err)
+		return nil, errors.Errorf("Read %q: %v", name, err)
```

Comment:
Since this is generated code, I don't think it's correct to replace `fmt` with `errors` in this file.